driver/pace/driver/grid.py

```diff
@@ -196,6 +199,82 @@ def get_grid(
         return damping_coefficients, driver_grid_data, grid_data
 
 
+@GridInitializerSelector.register("external")
+@dataclasses.dataclass
+class ExternalNetcdfGridConfig(GridInitializer):
+    """
+    Configuration for grid initialized from external data.
+    Input is from tile files generated by FRE-NCtools methods
```

Comment:
In addition to mentioning nctools, I recommend using the supergrid nomenclature and including an ascii art representation of two cells along with some info.

`X----X----X----X----X`
`|         |         |`
`X    X    X    X    X`
`|         |         |`
`X----X----X----X----X`